pyani_plus/workflows/snakemake_anib.smk

```diff
@@ -62,7 +62,11 @@ rule blastdb:
     output:
         "{outdir}/{genomeB}.njs",
     shell:
-        "chronic {params.makeblastdb} -in {input.genomeB} -input_type fasta -dbtype nucl -title {wildcards.genomeB} -out {wildcards.outdir}/{wildcards.genomeB}"
+        """
+        {params.makeblastdb} -in {input.genomeB} -input_type fasta -dbtype nucl \
```

Comment:
If we want to catch the log, rather than write to the filesystem , we may want to consider a wrapper function here.